packages/yoastseo/src/scoring/assessments/seo/MetaDescriptionKeywordAssessment.js

```diff
@@ -2,34 +2,34 @@ import { __, sprintf } from "@wordpress/i18n";
 import { merge } from "lodash";
 
 import Assessment from "../assessment";
-import { createAnchorOpeningTag } from "../../../helpers/shortlinker";
+import { createAnchorOpeningTag } from "../../../helpers";
 import AssessmentResult from "../../../values/AssessmentResult";
 
+/**
+ * @typedef {import("../../../languageProcessing/AbstractResearcher").default } Researcher
+ * @typedef {import("../../../values/").Paper } Paper
+ */
+
 /**
  * Assessment for checking the keyword matches in the meta description.
  */
-class MetaDescriptionKeywordAssessment extends Assessment {
+export default class MetaDescriptionKeywordAssessment extends Assessment {
 	/**
 	 * Sets the identifier and the config.
 	 *
 	 * @param {Object} [config] The configuration to use.
-	 * @param {number} [config.parameters.recommendedMinimum] The recommended minimum of keyword occurrences in the meta description.
 	 * @param {number} [config.scores.good] The score to return if there are enough keyword occurrences in the meta description.
-	 * @param {number} [config.scores.bad] The score to return if there aren't enough keyword occurrences in the meta description.
-	 * @param {string} [config.url] The URL to the relevant article on Yoast.com.
+	 * @param {number} [config.scores.bad] The score to return if there are no or too many keyword occurrences in the meta description.
+	 * @param {string} [config.urlTitle] The URL to the relevant article on Yoast.com to add to the title of the assessment in the feedback.
+	 * @param {string} [config.urlCallToAction] The URL to the relevant article on Yoast.com to add to the call to action in the assessment feedback.
 	 *
-	 * @returns {void}
 	 */
 	constructor( config = {} ) {
 		super();
 
 		const defaultConfig = {
-			parameters: {
-				recommendedMinimum: 1,
-			},
 			scores: {
 				good: 9,
-				ok: 6,
```

Comment:
The values I removed are not used anywhere (since this commit: https://github.com/Yoast/wordpress-seo/commit/e3f756b5baed6439ac29c5458f5d797a9f307ebd)